src/cargo_type.h

```diff
@@ -88,6 +88,9 @@ namespace CargoFilterCriteria {
 	static constexpr CargoID CF_NONE    = NUM_CARGO + 1; ///< Show only items which do not carry cargo (e.g. train engines)
 	static constexpr CargoID CF_ENGINES = NUM_CARGO + 2; ///< Show only engines (for rail vehicles only)
 	static constexpr CargoID CF_FREIGHT = NUM_CARGO + 3; ///< Show only vehicles which carry any freight (non-passenger) cargo
+
+	static constexpr CargoID CF_EMPTY      = NUM_CARGO + 4; ///< Show items without cargo (station list)
```

Comment:
Should this `CF_NO_RATING` or something, to reflect the change from cargo waiting to cargo rating?